File: Javascript/FindNumbersWithEvenNumberOfDigits.js
let findNumsWithEvenNumberOfDigits = function(nums) {  
    return nums.reduce((count, num) => {
        return count +  (num.toString().length % 2 === 0 ? 1 : 0);
    }, 0);
}
console.log(`Total: ${findNumsWithEvenNumberOfDigits([12, 345, 2, 6, 7896])}`);

File: LeetCode/Python/findNumbersWithEvenNumberOfDigits.py
#%%
from functools import reduce
def findNumsWithEvenNumberofDigits(nums):
    return reduce(lambda count, num: count + (1 if len(str(num)) % 2 == 0 else 0), nums, 0)
findNumsWithEvenNumberofDigits([12, 345, 2, 6, 7896])
# %%